lib/api-call.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const axios = require('axios');
+
+module.exports = {
+  getResponses: function(baseUrl, surveyID, endPoint, apiToken, apiTokenSecret) {
+    axios.get(baseUrl + '/' + surveyID + '/' + endPoint + '?api_token=' + apiToken + '&api_token_secret=' + apiTokenSecret)
```

Comment:
@chrisns's solution is also nice because it will fix a minor bug in your code. Specifically, anything that goes into a URL should first be escaped / URL encoded.